PeakLocalisations/Localisation/Blocks/ConfigureGoal.xcstrings

```diff
@@ -185,6 +245,12 @@
             "state" : "translated",
             "value" : "Add Resolution"
           }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Agregar Resolución"
```

Comment:
Añadir resolución sounds better to me but both are understandable

Reply:
Apple seems to use "Agregar" for Spain and "Añadir" for Latin America, so I just went with that too